projects/gnupg/fuzz_list.c

```diff
@@ -132,6 +135,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
         initialized = true;
     }
 
+    if (Size > MAX_LEN) {
+        // limit maximum size to avoid long computing times
+        Size = MAX_LEN;
```

Comment:
we usually return 0 for such cases.

Reply:
Ok.
Why is it better to return 0 ? (just to know)

Reply:
Won't catch overflows to the right since data is still allocated with the larger size.